MMM-voice.js

```diff
@@ -220,10 +222,51 @@ Module.register('MMM-voice', {
             MM.getModules().enumerate((module) => {
                 module.hide(1000);
             });
+            // tell other modules all hidden
+            this.sendNotification('NOW_ASLEEP','[]')
         } else if (notification === 'SHOW') {
             MM.getModules().enumerate((module) => {
                 module.show(1000);
             });
+            // tell other modules all shown
+            this.sendNotification('NOW_AWAKE')
+				} else if (notification === 'SLEEP_HIDE') {
+						// sleep by hiding (energyStar monitors)
+            let self=this;
```

Comment:
Check also the indentation of the files, they look all strange

Reply:
yes, I HATE that there are tabs.. or different treatment of tabs..